packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss

```diff
@@ -172,6 +172,11 @@
         display: grid;
         grid-template-rows: max-content 1fr;
 
+        @include desktop {
+            overflow-y: scroll;
+            max-height: calc(100vh - var(--header-nav-height) - 40px);
```

Comment:
Why 40px? Is it height of some block?

Reply:
It is distance from Overlay popup bottom to bottom of viewport. It looks nicer when there is spacing between overlay and bottom of the page.

Reply:
Would be great to move this 40px to variable with name which will make sense. Or at least leave comment about this value